source/ns_cmdline.c

```diff
@@ -1624,9 +1624,9 @@ int help_command(int argc, char *argv[])
         if (cmd_ptr) {
             cmd_printf("Command: %s\r\n", cmd_ptr->name_ptr);
             if (cmd_ptr->man_ptr) {
-                cmd_printf(cmd_ptr->man_ptr);
+                cmd_printf("%s\r\n", cmd_ptr->man_ptr);
```

Comment:
this is also behaviour change (because of extra line feeds) but shouldn’t cause any trouble - fine for me.